<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="previewTabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>600</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="cameraimages">
     <attribute name="title">
      <string>Camera images</string>
     </attribute>
     <widget class="QLabel" name="im1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="im2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="im4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>300</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="im3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>16</width>
        <height>571</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="load_images">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Calibration Images</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cap_images">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>50</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Capture Calibration Images</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_images">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>90</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Calibration Images</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>150</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>130</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Images name prefix:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="roi">
     <attribute name="title">
      <string>Extract features</string>
     </attribute>
     <widget class="QLabel" name="im2_roi">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="im4_roi">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>300</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="im1_roi">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="im3_roi">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>16</width>
        <height>571</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="extract_features">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>120</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Extract Features</string>
      </property>
     </widget>
     <widget class="QSlider" name="fd_thresh_slider">
      <property name="geometry">
       <rect>
        <x>629</x>
        <y>40</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="fd_thresh_label">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>40</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Feature detector threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>70</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Matcher confidence limit</string>
      </property>
     </widget>
     <widget class="QSlider" name="matcher_conf_slider">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>90</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>20</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>60</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="match_conf_label">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>90</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0.6</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stitched">
     <attribute name="title">
      <string>Stitched image</string>
     </attribute>
     <widget class="clickSignalQLabel" name="result">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>600</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>16</width>
        <height>571</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="stitch_images">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Stitch images</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_corners">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>50</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset corners</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="squared">
     <attribute name="title">
      <string>Squared</string>
     </attribute>
     <widget class="dragZoomQLabel" name="result_final">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>600</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>16</width>
        <height>571</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="square_arena">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Square arena</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_calib">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>50</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Save calibration</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>615</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>dragZoomQLabel</class>
   <extends>QLabel</extends>
   <header>dragzoomqlabel.h</header>
  </customwidget>
  <customwidget>
   <class>clickSignalQLabel</class>
   <extends>QLabel</extends>
   <header>clicksignalqlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>fd_thresh_slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>fd_thresh_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>861</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
